DAS/das/mqtt_wireless_logger.py

```diff
@@ -6,8 +6,9 @@
 import glob
 import re
 
+from mhp.topics import WirelessModule
```

Comment:
Same concern here, is it possible to import and rename to `WirelessModuleTopics`, or I think it may be a good idea to rename the class in `mhp.topics` directly. 
We also have a `WirelessModule` class in `wireless_module.py` and so having two classes with the same name can become confusing going forward, especially given that we would likely need to import `WirelessModule` from `mhp.topics` in the `wireless_module.py` file in the future to remove the literal topic strings used in that file.